parlai/scripts/flask_demo.py

```diff
@@ -0,0 +1,27 @@
+#import required libraries
+from flask import Flask, render_template, request
+from parlai.core.agents import create_agent_from_model_file
+
+
+# Flask constructor takes the name of
+#initialize flask app
+app = Flask(__name__)
+
+#import model from the model file can be pretrained or fine tuned
+blender_agent = create_agent_from_model_file("zoo:blender/blender_90M/model")

```

Comment:
Can you move this initialization into `main` somehow, so that the website builder doesn't do this?